tox.ini

```diff
@@ -17,13 +17,21 @@ envlist =
     docs
 
 [testenv]
-commands = pytest --lsof -ra {posargs:testing}
-passenv = USER USERNAME
+commands =
+    {env:_PYTEST_TOX_COVERAGE_RUN:} pytest --lsof -ra {env:_PYTEST_TEST_OPTS:} {posargs:testing}
+    coverage: coverage report -m --skip-covered
+setenv =
+    coverage: _PYTEST_TOX_COVERAGE_RUN=coverage run -a -m
```

Comment:
Why do we handle `_PYTEST_TOX_COVERAGE_RUN` and  `_PYTEST_TOX_EXTRA_DEP` both here and in the travis file?

Reply:
This is meant to allow for `tox -e py36-coverage`.
I've thought about/tried using only `-coverage`, but it is not trivial/feasible due to the many different testenvs.

Reply:
I see, OK, thanks